src/styles/base/_typography.scss

```
@@ -0,0 +1,31 @@
+body {
+    font-family: "Alan Sans";
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 1.5rem;
+}
+
+
```

fonti aw h1-h6 bo nakrdwa baw fontay dia?
```suggestion

h1, h2, h3, h4, h5, h6 {
    font-family: "Indie Flower", cursive;
}

```